src/compiler/crystal/tools/doc/markdown_doc_renderer.cr

```diff
@@ -38,13 +20,15 @@ class Crystal::Doc::MarkdownDocRenderer < Markdown::HTMLRenderer
       method = lookup_method @type, name, args
       if method
         text = method_link method, "#{method.prefix}#{text}"
-        @io << text
-        super
+        tag("code")
+        lit(text)
+        tag("/code")
         return
       end
     end
 
     # Check Type#method(...) or Type or #method(...)
+    # https://www.youtube.com/watch?v=SoIfcbsV13w
```

Comment:
srsly

Reply:
Oops, that was a joke and i intended to remove it before I pushed.

But the code below probably does need some refactoring...